README.md

```diff
@@ -0,0 +1,79 @@
+# Environment Variables Injector
+
+This extension is designed to help users export snap options configured with the [snap set command](https://snapcraft.io/docs/configuration-in-snaps) into environment variables.
+
+Additionally, this extension supports environment files, which we refer to as `envfile` for brevity.
+
+To prevent conflicts with other snap configurations and to maintain some level of isolation between apps, the following rules apply:
+
+* **Global:** Environment variables are visible to all apps within the snap that use this extension.
+* **Local:** Environment variables are visible only to the specific `app` defined during the snap set command.
+
+|        | Snap Configuration Option      | Environment Variable |
+|--------|--------------------------------|----------------------|
+| Global | env.<key>=<value>              | <KEY>=<value>        |
+| Local  | apps.<app>.env.<key>=<value>   | <KEY>=<value>        |
+
+**Examples**
+
+Assuming `my-snap` is the name of the snap that uses the env-injector extension:
+
+* `sudo snap set <my-snap> env.default_port=8080` will be mapped to `DEFAULT_PORT=8080` globally.
+
+* `sudo snap set <my-snap> apps.server.env.endpoint="/v2/store"` will be mapped to `ENDPOINT="/v2/store"` and will be visible only to the `server` app.
+
+For environment files, where `path` refers to the `envfile` path:
```

Comment:
`envfile` is not an alias for environment files. It is part of the snap option key for environment file / env file path.

`envfile` - > env file

Reply:
Sure, fixed! On 033f468